framework/db/db.py

```diff
@@ -149,20 +147,16 @@ def CreateEngine(self, BaseClass):
         except ValueError as e:  # Potentially corrupted DB config.
             self.error_handler.FrameworkAbort(
                 'Database configuration file is potentially corrupted. '
-                'Please check ' + self.config.FrameworkConfigGet('DATABASE_SETTINGS_FILE') + '\n'
-                '[DB] ' + str(e))
+                'Please check %s\n[DB] %s' % (dself.config.FrameworkConfigGet('DATABASE_SETTINGS_FILE'), str(e)))
         except KeyError:  # Indicates incomplete db config file
             self.error_handler.FrameworkAbort(
-                "Incomplete database configuration settings in "
-                "" + self.config.FrameworkConfigGet('DATABASE_SETTINGS_FILE'))
+                "Incomplete database configuration settings in %s" %
+                self.config.FrameworkConfigGet('DATABASE_SETTINGS_FILE'))
         except exc.OperationalError as e:
-            self.error_handler.FrameworkAbort(
-                "[DB] " + str(e) + "\nRun scripts/db_run.sh to start/setup db")
+            self.error_handler.FrameworkAbort("[DB] %s\nRun scripts/db_run.sh to start/setup db" % str(e))
 
```

Comment:
https://github.com/hTrap/owtf/blob/pep8-fixes/framework/db/db.py#L150 - error!
It is `self` not `dself` :P